### PR TITLE
save file locally

### PR DESCRIPTION
**Pull request by madison-kerndt**

### CODE REVIEW

File: app/components/App.js [hunk 1]
```diff
@@ -101,6 +101,13 @@ export default class App extends React.Component {
     this.startNewNote();
   }
 
+  saveLocal() {
+    const { remote } = require('electron');
```
Comment (Casey1449) [inline app/components/App.js]: Never used 'require' inside a component before...bold move cotton.

File: app/main.js [hunk 2]
```diff
@@ -28,4 +30,18 @@ app.on('ready', function() {
   mainWindow.on('closed', function() {
    mainWindow = null;
   });
+
+
+  const saveLocal = app.saveLocal = (win, content) => {
+    const file = dialog.showSaveDialog(win, {
+      title: 'Save File',
+      defaultPath: app.getPath('desktop'),
+      buttonLabel: 'Save your note 🗒',
```
Comment (Casey1449) [inline app/main.js]: beautiful

File: app/main.js [hunk 3]
```diff
@@ -28,4 +30,18 @@ app.on('ready', function() {
...
+      filters: [
+        {name: 'All Files', extensions: ['*']}
```
Comment (Casey1449) [inline app/main.js]: We should limit the extensions later if we have time

File: app/main.js (hunk 2 above)
Comment (madison-kerndt) [inline app/main.js]: On line 38 do you think that is the right default file path?

**Pull request merged**